.github/workflows/dockerhub-pr.yml

```diff
@@ -0,0 +1,24 @@
+name: DockerHub (PR)
```

Comment:
Do we really need to push to dockerhub for each PR? Seems a bit overkill to me.

Reply:
I also thought it'd be overkill, but I also realized that by building an image out of a PR, any reviewer can very quickly test out that build by simply firing up a local instance that refers to that image.

I myself am fine with either having this or not, so let's also wait to hear from @Robert-Steiner as he's also a reviewer.

Reply:
Yes, I think that could be a little too much for now. But I like the idea of fetching an image so that I can test it. Maybe we can use the [Pull Request Comment Trigger| Github action](https://github.com/marketplace/actions/pull-request-comment-trigger) to run this job on demand. wdyt?

Reply:
I think that's a neat idea! I can apply that change.

Reply:
@Robert-Steiner I've been for the past 2h banging my head against the wall. The problem is: although I also like very much the idea of triggering the build upon a keyword on a comment posted to a PR, achieving that is proving to be tricky because the `actions/checkout` will insist on using `master` branch when the build is triggered by the comment. I will need to do a ridiculous workaround querying GitHub's API just so I can get the branch in which the PR is based.
I'm putting some more thought into it and thinking of alternatives.

Reply:
Yeah we can find a solution later 🙂 I'm already happy that we have an image for each merge

Reply:
I'm insisting on it a bit, let's see...

Reply:
I'll be damned, it works! :rocket: